Cbc/src/CbcModel.hpp

```diff
@@ -448,10 +450,17 @@ class CbcModel {
     */
   void analyzeObjective();
   /** Returns postProcessed solution in solver(called from event handler)
+   * or nullptr.
      Normally used for integer solution (not really tested otherwise)
-    solutionType 1 is best integer so far, 0 is current solution 
+    solutionType 1 is best integer so far, 0 is current solution
     (may not be integer) */
   const OsiSolverInterface *postProcessedSolver(int solutionType = 1);
+  /** Returns postProcessed solution in solver(called from event handler)
+   * or solution of the original model if that has not been preprocessed.
+     Normally used for integer solution (not really tested otherwise)
+    solutionType 1 is best integer so far, 0 is current solution
```

Comment:
I would prefer an enum, which would make the code later easier to read.

Reply:
Sure an enum is clearer, added. However this interface just builds on the existing function postProcessedSolver(int ), so I also changed it. And the couple of places where both are used. Including example inc.cpp.